migrations/20200513_01_kPWNp-create-base-tables.py

```diff
@@ -0,0 +1,56 @@
+# 20200513_01_kPWNp-create-base-tables.py
+
+from yoyo import step
+
+__depends__ = {}
+
+steps = [
+    step(
+        'CREATE TYPE team_method AS ENUM(\'captains\', \'autobalance\', \'random\');',
+        'DROP TYPE team_method;'
+    ),
+    step(
+        'CREATE TYPE captain_method AS ENUM(\'volunteer\', \'rank\', \'random\');',
```

Comment:
Can you use `"` so that you don't have to escape the quotes?

Reply:
Yeah I will change that.

Reply:
So apparently double quotes aren't for strings in PostgreSQL